qiskit_ibm/credentials/credentials.py

```diff
@@ -51,13 +52,14 @@ def __init__(
             services: Optional[Dict] = None,
             access_token: Optional[str] = None,
             preferences: Optional[Dict] = None,
-            default_provider: Optional[HubGroupProject] = None
+            default_provider: Optional[HubGroupProject] = None,
     ) -> None:
         """Credentials constructor.
 
         Args:
-            token: IBM Quantum Experience API token.
-            url: IBM Quantum Experience URL.
+            token: IBM Quantum API token.
+            url: IBM Quantum URL.
+            api_url: IBM Quantum URL.
```

Comment:
`api_url` always will be `https://auth.quantum-computing.ibm.com/api` whereas `url` is overwritten with the service url `https://api-qcon.quantum-computing.ibm.com/...` once Credentials object is initialized. `api_url` is read from the first provider's credentials (from _providers class variable) when the users checks the active account. (Since we have removed the `_credentials` class level variable)